src/main/java/baseball/result/Result.java

```diff
@@ -0,0 +1,54 @@
+package baseball.result;
+
+import baseball.number.GameNumber;
+
+public class Result {
+    private int strikeCount;
+    private int ballCount;
+    public Result() {
+        this.strikeCount = 0;
+        this.ballCount = 0;
+    }
+
+    public void calculateResult(GameNumber seedNumber, GameNumber userNumber) {
+        calculateStrikeCount(seedNumber, userNumber);
+        calculateBallCount(seedNumber, userNumber);
+    }
+
+    private void calculateStrikeCount(GameNumber seedNumber, GameNumber userNumber) {
+        int strikeCount =0;
+        String seedValue = seedNumber.getValue();
+        String userValue = userNumber.getValue();
+
+        for (int index = 0; index < seedValue.length(); index++) {
+            if (userValue.charAt(index) == seedValue.charAt(index)) {
+                strikeCount++;
+            }
+        }
+        this.strikeCount = strikeCount;
+    }
+
+    private void calculateBallCount(GameNumber seedNumber, GameNumber userNumber) {
+        String seedValue = seedNumber.getValue();
+        String userValue = userNumber.getValue();
+        int ballCount = (int) seedValue.codePoints().mapToObj(digit -> (char)digit)
+                            .filter(digit -> userValue.indexOf(digit) != -1).count();
```

Comment:
codePoints와 같은 stream의 경우 가독성을 위해 stream이 시작되고 나서 개행을 해주곤 합니다 🙂
```suggestion
        int ballCount = (int) seedValue.codePoints()
                .mapToObj(digit -> (char) digit)
                .filter(digit -> userValue.indexOf(digit) != -1)
                .count();
```